strapi/strapi
# [Custom Fields] Add tabs to attribute modal

### What does it do?

- Adds tabs to the attribute creation modal
- One tab for the default Strapi types, and one tab for Custom Fields
- Displays "Coming Soon" on the custom fields tabs

### Why is it needed?

To accommodate custom fields in the attribute create modal

### How to test it?

- Go to to any content-type or component and click "Add another field", you should see the tabs
- The "Default" tab should be selected by default
- Click the "custom" tab, you should see "Coming soon" text
